src/ShellProgressBar/ShellProgressBar.csproj

```diff
@@ -14,6 +14,10 @@
     <PackageIconUrl>https://raw.github.com/Mpdreamz/shellprogressbar/master/src/nuget-icon.png</PackageIconUrl>
     <PackageTags>console;shell;terminal;progress;progressbar</PackageTags>
   </PropertyGroup>
+  <PropertyGroup>
+    <DocumentationFile>D:\Code\shellprogressbar\src\ShellProgressBar\ShellProgressBar.xml</DocumentationFile>
```

Comment:
Adding an absolute path to the documentation file is a bad idea since the path will not be the same across different machines. You should use `<GenerateDocumentationFile>true</GenerateDocumentationFile>` instead and the documentation file will automatically be generated in the correct location.

Reply:
++ massive oversight on my end :) added `<GenerateDocumentationFile>` in ac6dc22f284a99e4bc76c6b4463841e3718a5998